src/renderer/src/pages/home/index.tsx

```
@@ -1,16 +1,19 @@
   return (
-    <div>
-      홈 페이지 입니다. 다들 방가방가 ping 한번 넣어봄.
-      <div className="action">
-        <a target="_blank" rel="noreferrer" onClick={ipcHandle}>
-          Send IPC
-        </a>
-      </div>
-    </div>
+    <StyledHome>
+      <Chart />;
+    </StyledHome>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)